managed/utils/envvars/parser.go

```diff
@@ -56,13 +56,13 @@ func (e InvalidDurationError) Error() string { return string(e) }
 // In case of error, the docker run terminates.
 // Short description of environment variables:
 //   - PATH, HOSTNAME, TERM, HOME are default environment variables that will be ignored;
-//   - DISABLE_UPDATES is a boolean flag to enable or disable pmm-server update;
-//   - DISABLE_TELEMETRY is a boolean flag to enable or disable pmm telemetry (and disable STT if telemetry is disabled);
-//   - DISABLE_ALERTING disables Percona Alerting;
-//   - METRICS_RESOLUTION, METRICS_RESOLUTION_MR, METRICS_RESOLUTION_HR, METRICS_RESOLUTION_LR are durations of metrics resolution;
-//   - DATA_RETENTION is the duration of how long keep time-series data in ClickHouse;
-//   - ENABLE_AZUREDISCOVER enables Azure Discover;
-//   - ENABLE_RBAC enables Access control;
+//   - PMM_DISABLE_UPDATES is a boolean flag to enable or disable pmm-server update;
+//   - PMM_DISABLE_TELEMETRY is a boolean flag to enable or disable pmm telemetry (and disable STT if telemetry is disabled);
+//   - PMM_DISABLE_ALERTING disables Percona Alerting;
+//   - PMM_METRICS_RESOLUTION, PMM_METRICS_RESOLUTION_MR, PMM_METRICS_RESOLUTION_HR, PMM_METRICS_RESOLUTION_LR are durations of metrics resolution;
+//   - PMM_DATA_RETENTION is the duration of how long keep time-series data in ClickHouse;
+//   - PMM_ENABLE_AZURE_DISCOVER enables Azure Discover;
+//   - PMM_ENABLE_RBAC enables Access control;
```

Comment:
@idoqo @BupycHuk 
Guys, do we want this to be `PMM_ENABLE_ACCESS_CONTROL` going forward? That seems to be in line with the other migration, where we rename the gRPC service "Roles" to `AccessControl`.